From fe1dc848aa75d567e03855cef895917f6121067e Mon Sep 17 00:00:00 2001
From: Angie Chinchilla <angie.v.chinchilla@intel.com>
Date: Fri, 6 Apr 2012 18:06:56 +0300
Subject: [PATCH 71/75] Clarify comment in wpa_supplicant/defconfig

I think this comment, as originally stated, says the opposite of
what is meant. Reword to help the next guy.

Signed-hostap: Angie Chinchilla <angie.v.chinchilla@intel.com>
(cherry picked from commit 540609c55fd4b08e85029b5907188bc748ee59b2)
---
 wpa_supplicant/defconfig |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/wpa_supplicant/defconfig
+++ b/wpa_supplicant/defconfig
@@ -411,7 +411,7 @@ CONFIG_PEERKEY=y
 # This tracks use of memory allocations and other registrations and reports
 # incorrect use with a backtrace of call (or allocation) location.
 #CONFIG_WPA_TRACE=y
-# For BSD, comment out these.
+# For BSD, uncomment these.
 #LIBS += -lexecinfo
 #LIBS_p += -lexecinfo
 #LIBS_c += -lexecinfo
@@ -420,7 +420,7 @@ CONFIG_PEERKEY=y
 # This enables use of libbfd to get more detailed symbols for the backtraces
 # generated by CONFIG_WPA_TRACE=y.
 #CONFIG_WPA_TRACE_BFD=y
-# For BSD, comment out these.
+# For BSD, uncomment these.
 #LIBS += -lbfd -liberty -lz
 #LIBS_p += -lbfd -liberty -lz
 #LIBS_c += -lbfd -liberty -lz
